package.json

```diff
@@ -4,24 +4,29 @@
   "description": "The documentation content home for https://docs.quantum.ibm.com.",
   "author": "Qiskit Development Team",
   "license": "Apache-2.0",
+  "type": "module",
+  "files": [
+    "dist"
+  ],
   "scripts": {
+    "prepare": "tsc",
```

Comment:
`prepare` is a special script. VCS requirements run `prepare` after Git cloning the repo.